community/standards/julia_stewards.md

```diff
@@ -0,0 +1,52 @@
+
+## Stewards of the Community Standards
+
+All members of the Julia community are expected to abide by the community standards and encourage others to do so.
+To ensure that some group has the obligation and authority to cope with conflicts, a group known as the "Julia Stewards" has been created.
+The current membership of this group is:
+
+ - Milan Bouchet-Valat
+ - Simon Byrne
+ - Tim Holy
+ - Katherine Hyatt
+ - Steven G. Johnson
+ - Stefan Karpinski
+ - Viral Shah
+
+As needed, this group can be contacted at [stewards@julialang.org](mailto:stewards@julialang.org).
+Other than the committee listed above, there are no other recipients of emails sent to this address, and **all communications shall be treated confidentially**.
+
+
+The guidelines governing the Julia Stewards' actions are described below.
+
+## Complaint Resolution
+
+**No one shall be retaliated against for good-faith participation in a complaint**.
+In general, violations of the community standards will generate one or more of the following responses:
+
+
+1. Informal feedback, privately and/or (for public violations) in the forum where a violation appeared, with the goal of encouraging good-faith contributions and mutual understanding while making offenders aware of the problem and preventing future incidents. Where possible, good intentions of the participants should be assumed.
+2. Gross online violations of community standards may result in immediate deletion of the offending comments, coupled with informal feedback.
+3. For severe cases, especially persistent, disruptive violations despite repeated warnings and other feedback, a formal investigation may be convened by the stewards.
+   * A consensus committee of 3+ disinterested stewards (possibly including non-steward members) shall investigate the complaint, communicating this with both the complainant (if any) and the respondent (alleged violator).
+   * The investigation shall be as confidential as possible, and this expectation shall be communicated to all participants. Identities and testimonies of participants shall, where possible, be known only to the committee.
+   * The committee shall provide a private written report detailing its findings and recommendation to the stewards, complainant, and respondent. If a violation has been found, recommendations may include bans and other alterations of online privileges, or in less severe cases may request apologies and other informal resolutions.
+   * The conclusions of the investigative committee are final and cannot be appealed except in cases of gross misconduct or major factual errors.
+
+## Bans
+
+The public Julia mailing lists (e.g. julia-users) and GitHub pages (e.g. issues and pull requests) are normally open forums for all good-faith contributors.
```

Comment:
Better still, make it a link to the google group: https://groups.google.com/forum/#!forum/julia-users.